samples/subsys/usb/mass/Kconfig

```diff
@@ -77,22 +74,6 @@ if NORDIC_QSPI_NOR
 config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
 	default 4096
 
-config DISK_FLASH_DEV_NAME
-	default "MX25R64" if "$(dt_nodelabel_enabled,mx25r64)"
-	default "GD25Q16" if "$(dt_nodelabel_enabled,gd25q16)"
```

Comment:
Before this PR, flashdisk.c is using CONFIG_DISK_FLASH_DEV_NAME, but with it, this is moving to any fixed-partitions nodes, which I think will include the ones on the SoC flash too. Is that intentional? Sorry, I'm not familiar with the FS layer and I'm trying to understand the goals of this PR.